backend/app/infrastructure/__init__.py

```diff
@@ -56,20 +60,62 @@ def __getattribute__(self, name):
         return attr
 
 
+def lookup_max_time():
+    assert max_time is not None
+    return max_time
+
+
+def is_graph_error_permanent(error):
```

Comment:
and for now the function always returns False, shouldn't it be changed?

Reply:
I'm leaving this thing in place so we can collect logs on how exactly this exception carries strings that we should match on.

Reply:
okay, so i don't know if it won't be a better option to change print to ```app.logger.debug``` (we use it in other places of code) to make sure we'll be able to grab the logs from testing environment on grafana and just to be consist with the approach

Reply:
+1